qgispluginci/translation_clients/transifex.py

```diff
@@ -133,7 +133,10 @@ def get_translation(
             resource=self.get_resource(), language=language
         )
 
-        translated_content = requests.get(url).text
+        r = requests.get(url)
+        # transifex returns None encoding and the apparent_encoding is Windows-1254 what leads to malformed result strings. So we set the encoding hardcoded to utf-8.
+        r.encoding = "utf-8"
```

Comment:
can you maybe do a test if it's null? (so maybe we catch something if Transifex fixes this?)

@Guts are you ok with this?

Reply:
Done.

Reply:
Thank you, I find this more suitable.